src/js/pages/Index.js

```diff
@@ -140,14 +144,55 @@ var Index = React.createClass({
     );
   },
 
+  handleWindowResize(event) {
+    // TODO: trottle method using RXJS
```

Comment:
Please remove the TODOs, if we end up not doing this they will become stale as so many todos in our app did already 🙈

Reply:
What would you think about only having TODOS that have tickets attached to it?